.pre-commit-config.yaml

```diff
@@ -69,7 +69,8 @@ repos:
   rev: f7f304cb40596e4706be27ca44d1401396a97420
   hooks:
   - id: codespell
-    entry: tools/codespell.py --ignore-regex "\\W(?:(?:m_)?p(?=[A-Z])|m_(?=\\w))" --files
+    # ignore-regex is substituted by ' '
+    entry: tools/codespell.py --ignore-regex "\\W(?:pInOut|m_p*(?=[A-Z])|m_(?=\\w)|pp*(?=[A-Z])|k(?=[A-Z])|s_(?=\\w))" --files
```

Comment:
Can we use the `-L` flag to ignore `InOut`? The regex is already complicated IMHO.

Reply:
By the way, I think we should use `-w`. That would make it easier to apply the suggestions.

Reply:
Unfortunately the -L flag does not work. I guess it is a odd combination of p prefix and short cammel case words.

I don't think it is a good idea to use the -w fix. Like with clang format, we are used to trust the code checker. However we cannot trust codespell to this degree. From this point of view I am in favor to keep typos compared to the wrong corrected word that changes the sense of a sentences.

Reply:
Ok